dask-gateway/dask_gateway/gateway.yaml

```diff
@@ -19,3 +19,9 @@ gateway:
   cluster:
     options: {}         # Default options to use when calling ``new_cluster`` or
                         # ``cluster_options``. Values may be template strings.
+                        # authentication class above.
+
+  proxy_config:         # Values for a config dictionary for CurlAsyncHTTPClient
+    proxy_host: null    # Local proxy host
+    proxy_port: null    # Local proxy port
+    ca_certs: null      # Custom certificates to use (available from `certifi.where()`)
```

Comment:
Since both http clients can accept many parameters, perhaps we should instead have the following fields:

```
httpclient:
  type: null         # The kind of http client to use. One of {"curl", "simple"}. If not provided, curl is used if pycurl is installed, otherwise simple is used.
  defaults: null     # A mapping of default parameters to use on every request
```

This allows supporting any of the possible parameters, without requiring them to be exposed directly in the config file.

Reply:
Sounds good.  I'll put that together.